README.md

```diff
@@ -60,7 +60,11 @@ This repo is meant to provide French translation for the [reactjs.org](https://r
 
 ## Translation
 
+<<<<<<< HEAD
 If you are interesting in translating `reactjs.org`, in another language please see the current translation efforts at [isreactreadyyet.com](https://www.isreactreadyyet.com).
+=======
```

Comment:
@tesseralis is it me or did you just leave conflict markers in your commit? Sounds like you could use pre-commit check hooks 😉

Reply:
@tesseralis  Also this should say "interested", not "interesting"

Reply:
@tdd the pre-commit markers are meant to be there! These are auto-generated by a bot and it's up to the language maintainers to fix the conflicts themselves (say, if a page that's already been translated has been updated).

Also gdi I did make a spelling error >_<